source/validate.cpp

```diff
@@ -117,6 +119,40 @@ void DebugInstructionPass(ValidationState_t& _,
   }
 }
 
+// Parses OpExtension instruction and registers extension.
+void RegisterExtension(ValidationState_t& _,
+                       const spv_parsed_instruction_t* inst) {
+  const std::string extension_str = libspirv::GetExtensionString(inst);
+  Extension extension;
+  if (!ParseSpvExtensionFromString(extension_str, &extension)) {
+    // The error will be logged in the ProcessInstruction pass.
+    return;
+  }
+
+  _.RegisterExtension(extension);
+}
+
+// Parses the beginning of the module searching for OpExtension instructions.
+// Registers extensions if recognized. Returns SPV_REQUESTED_TERMINATION
+// once an instruction which is not SpvOpCapability and SpvOpExtension is
+// encountered (according to the SPIR-V spec extensions are declared after
```

Comment:
The parenthetical remark is a full sentence.  So end the previous sentence with a period, and captializze "According..."